src/main/java/org/checkerframework/checker/https/HTTPSAnnotatedTypeFactory.java

```diff
@@ -46,4 +48,16 @@ private void setStringToHTTPS(Tree tree, AnnotatedTypeMirror type){
             }
         }
     }
+
+    private void checkConcatenatedString (Tree tree, AnnotatedTypeMirror type) {
```

Comment:
Please add JavaDoc for this method that explains the type rule that it implements.